definitions/ext-dell_compellent/summary_metrics.yml

```diff
@@ -0,0 +1,7 @@
+diskCount:
```

Comment:
As we said in the previous PR can you use the golden metrics her too? that will make the migration much easier for everyone.

Reply:
done